### ISSUE TITLE
[Bug] Pasting formatted text removes spaces

### ISSUE THREAD

**Issue by Astridchan** [Floens/Clover#679]
When you paste formatted text containing bolded or italicised words (say from an online article) into the Clover text box, spaces get removed before and after the formatted words in the actual post that gets posted on 4chan. Here is an example:
Text copied and pasted to Clover text box:

plain text _italic text_ plain text **bold text** plain text _italic text_ plain text **bold text** plain text _italic text_ plain text **bold text** plain text _italic text_ plain text **bold text**

What gets posted on 4chan:
plain textitalic textplain textbold textplain textitalic textplain textbold textplain textitalic textplain textbold textplain textitalic textplain textbold text

![Screenshot_2019-04-14-21-06-37-909_org floens chan](https://user-images.githubusercontent.com/40862101/56097160-6cfe8d00-5ef9-11e9-8a8f-1d98770a46f6.png)

I would use the "paste as plain text" option, but the Android phone I'm using currently does not have that option available in the text selection menu. The reply text box on the 4chan website seems to automatically turn all pasted text into plain text, maybe Clover should do the same?

**Comment by K1rakishou** [comment]
Can't reproduce.
![234234](https://user-images.githubusercontent.com/10332458/58372431-c52ba480-7f25-11e9-8f5f-269b84045c45.png)

**Comment by Astridchan** [comment]
> Can't reproduce.

I tried it on another Android device and the same thing happens. So I can confirm this bug happens on my Pocophone F1 running Android 9.0 and Nvidia Shield K1 running Android 7.0. Could the keyboard app be the issue? Both devices are using Gboard. I guess I'll download another keyboard and see what happens.
 
EDIT: Tried another keyboard app (Hacker's keyboard), same thing happens.
